### PR TITLE
add bitrate type field to API

### PR DESCRIPTION
**Pull request by badhrinathpa**

### CODE REVIEW

File: pfcpiface/bess.go [hunk 1]
```diff
@@ -1146,6 +1150,12 @@ func (b *bess) addSliceMeter(ctx context.Context, done chan<- bool, meterConfig
 			ebs = 0 // Unused
 		}
 
+		log.Println("uplink slice :")
+		log.Println("cir : ", cir)
+		log.Println("pir : ", pir)
+		log.Println("cbs : ", cbs)
+		log.Println("pbs : ", pbs)
+		log.Println("ebs : ", ebs)
```
Comment (krsna1729) [inline pfcpiface/bess.go:1158]: Single line log.Traceln(). Same for below.

Comment (badhrinathpa) [reply pfcpiface/bess.go]: done

File: pfcpiface/web-service.go [hunk 2]
```diff
@@ -96,13 +98,46 @@ func sendHTTPResp(status int, w http.ResponseWriter) {
...
+			val = int64(mbr) * MB
+		}
+	case "Gbps":
+		{
+			val = int64(mbr) * GB
+		}
+	case "bps":
+		fallthrough
+	default:
+		return mbr
```
[the remark this answers is not in the document]
Comment (badhrinathpa) [reply pfcpiface/web-service.go]: done

**Comment by badhrinathpa** [comment]
retest this please

**Comment by thakurajayL** [comment]
retest this please

**Comment by sindack** [comment]
test container

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**